### PR TITLE
Add routing

### PR DESCRIPTION
**Pull request by aziz512**
This pull request proposes adding routes to the application and is created as a response to #2

### CODE REVIEW

**Comment by prestonvanloon** [comment]
This doesn't change the path as you change slides.

**Comment by aziz512** [comment]
@prestonvanloon I updated the code and now it does =) Please take a look.

**Comment by kirjs** [comment]
As discussed in chat let's decouple routing from the demo and make it an optional module.

File: src/app/demo/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (kirjs) [inline src/app/demo/app.component.ts]: router outlet is still outside of the slides lib?

File: src/app/routes.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//routes generator
+export function getSlidesRoutes(SlidesComponent, route?:string){
```
Comment (kirjs) [inline src/app/routes.ts]: call it rootRoute or something? 
Are you sure we need this at all? 
Doesn't it do the same thing as forChild?

**Comment by aziz512** [comment]
44a815b435df21a87c581ef1e16d25c6852e8a2b
e1be24df8952dc7036ee915c254e710896caeb9e

**Pull request closed without merging**